cmd/openshift-install/create.go

```diff
@@ -217,6 +221,7 @@ func newCreateCmd() *cobra.Command {
 			return cmd.Help()
 		},
 	}
+	cmd.PersistentFlags().BoolVar(&createClusterOpts.hideLoginInfo, "hide-login-info", false, "Do not print login info after the installation is complete")
```

Comment:
The description of this flag could use some cleanup given the changes to what is actually printed.

Reply:
@staebler could you please check if it's fine now? Thanks